output.c

```diff
@@ -42,6 +43,25 @@
 #include "xwayland.h"
 #endif
 
+static void
+update_output_manager_config(struct cg_server *server)
```

Comment:
We need to call this also in `handle_output_commit` in case of `event->committed & (WLR_OUTPUT_STATE_TRANSFORM | WLR_OUTPUT_STATE_SCALE)`, and I guess in `handle_output_mode` as well.

Reply:
I amended this to your branch :)